tests/api_tests/nodes/test_views.py

```diff
@@ -1352,7 +1467,7 @@ def test_filtering_on_invalid_field(self):
         assert_equal(res.status_code, 400)
         errors = res.json['errors']
         assert_equal(len(errors), 1)
-        assert_equal(errors[0]['detail'], 'Query string contains an invalid filter.')
+        assert_equal(errors[0]['detail'], "'invalid' is not a valid field for this endpoint.")
```

Comment:
I really like friendly and helpful, but *apparently* that's dangerous. As long as it's cleaned, it should be fine; it's not like we can avoid showing user input always, we just need to make sure that whatever it is is definitely cleaned.